airflow-core/src/airflow/ui/src/pages/DagRuns.tsx

```diff
@@ -187,6 +195,16 @@ export const DagRuns = () => {
     },
   );
 
+  // Client-side filtering for triggering_user_name (backend doesn't support this filter yet)
+  const filteredDagRuns =
+    data?.dag_runs.filter((run) => {
+      if (filteredTriggeringUserName === null || filteredTriggeringUserName === "") {
+        return true;
+      }
+
+      return run.triggering_user_name?.toLowerCase().includes(filteredTriggeringUserName.toLowerCase());
+    }) ?? [];
```

Comment:
Remove this please, this will be plain wrong because of backend pagination. It would be better to not add the filtering for now, and add the backend feature in a separate PR.